src/status_im/common/promesa/utils.clj

```diff
@@ -1,4 +1,4 @@
-(ns status-im.common.promesa.core
+(ns status-im.common.promesa.utils
```

Comment:
this util looks more general, so it shouldn't  be under status-im, it could go into utils ns